java/ql/test/stubs/unboundid-ldap-4.0.14/com/unboundid/ldap/sdk/Filter.java

```diff
@@ -0,0 +1,13 @@
+package com.unboundid.ldap.sdk;
+
+public class Filter {
+  public static Filter create(java.lang.String filterString) throws LDAPException { return null; }
+
+  public static Filter createNOTFilter(Filter notComponent) { return null; }
+
+  public static Filter createEqualityFilter(java.lang.String attributeName, java.lang.String assertionValue) { return null; }
+
+  public static java.lang.String encodeValue(java.lang.String value) { return null; }
+
+  public void toNormalizedString(java.lang.StringBuilder buffer) { }
+}
```

Comment:
This is the root cause for the second false negative (`uBadFilterCreateToString` argument not detected) - `toString` method definition is missing.
Adding `toString` method returning `null` fixes the problem.